free-1.md

```diff
@@ -4,40 +4,120 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A CAOS (Computer Aided Organic Synthesis) DSL.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Currently, chemists have one of three ways in which they can predict how a 
+reaction will occur, given certain molecules and conditions.
+
+1. Map it out by hand
```

Comment:
Having taken frosh Chem, I can get behind the idea of making a tool that saves you from having to map these reactions out by hand.